regression/Makefile

```diff
@@ -16,6 +16,7 @@ DIRS = ansi-c \
        invariants \
        strings \
        jbmc-strings \
+#       smt2_solver \
```

Comment:
What's the effect of having a commented-out line ending in a `\` in a block of lines that end in `\`? (It certainly confuses GitHub's syntax highlighting, but possibly also `make`.)

Reply:
Removed